keyboards/latinpad/config.h

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 latincompass
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.*/
+
+#pragma once
+
+#include "config_common.h"
+#define VENDOR_ID       0x4444 // "ww" = ww
+#define PRODUCT_ID      0x5350 // "ri" = rich
```

Comment:
These comments dont match the ascii hex representations as they are kinda just the snagpad values

```suggestion
#define VENDOR_ID       0x6C63 // "lc" = latincompass
#define PRODUCT_ID      0x6C70 // "lp" = latinpad
```

Reply:
Yes, I used part of the code for this keyboard